docker-compose.yml

```diff
@@ -24,7 +24,8 @@ services:
       - RABBITMQ_DEFAULT_VHOST=vhost
 
   api:
-    build: ./api
+    build:
+      context: ./api
     ports:
```

Comment:
Uit nieuwsgierigheid, waarom heb je deze change gedaan?

Reply:
Volgens mij was deze change uiteindelijk niet nodig, maar ik heb kennelijk niet alles teruggedraaid.

Eerst had ik de docker-compose en project structuur helemaal veranderd zodat die meer binnen de richtlijnen van onze andere projecten vielen. Maar ik besloot dat het beter was om de changes minimaal te houden. Ik denk dat het wel preferabel zou zijn om alle projecten gelijk te hebben, maar het leek me niet super handig om nu de hele structuur om hoop de gooien (als we dit project long-term zouden moeten onderhouden waarschijnlijk wel). Als een minimum wou ik dezelfde make targets hanteren waardoor het iets makkelijker wordt om mee te werken.